modal/cli/profile.py

```diff
@@ -53,7 +54,20 @@ async def list(json: Optional[bool] = False):
         content = ["•" if active else "", profile, workspace]
         rows.append((active, content))
 
+    env_based_workspace: Optional[str] = None
+    if "MODAL_TOKEN_ID" in os.environ:
```

Comment:
shouldn't we just use `config.get` to grab the tokens just like the server url?

Reply:
That will tell you the token that is being used, but what we want to know here is whether that is different from the token associated with the “active profile”.

Reply:
Ah, so this is specifically for when a user specifies TOKEN_ID and TOKEN_SECRET as env vars but has an active profile which is otherwise associated with another workspace? I guess we could skip showing the message if the env-var workspace and the profile workspace are the same?

Reply:
Yeah exactly. It could also make sense to only show this when they disagree. Maybe one reason to show this kind of message even when they agree is that it might help people problem solve faster if they subsequently get themselves into a state when they disagree. Maybe that suggests that `modal profile activate` should also provide some sort of feedback if you activate a profile with a workspace that won't actually get used because the environment variable tokens will override it.